checkers: flagName segfault on bufio/bufio_test.go

```
flagName: error: runtime error: invalid memory address or nil pointer dereference
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x79b7bd]

goroutine 278 [running]:
github.com/go-lintpack/lintpack/linter/lintmain/internal/check.(*linter).checkFile.func1.1(0xc0000ebec0, 0xc0001ad500)
        /home/neverik/go/src/github.com/go-lintpack/lintpack/linter/lintmain/internal/check/check.go:136 +0x167
panic(0x807560, 0xa97990)
        /usr/local/go/src/runtime/panic.go:513 +0x1b9
go/types.(*Package).Path(...)
        /usr/local/go/src/go/types/package.go:30
github.com/go-critic/go-critic/checkers.isStdlibPkg(...)
        /home/neverik/go/src/github.com/go-critic/go-critic/checkers/utils.go:13
github.com/go-critic/go-critic/checkers.(*flagNameChecker).VisitExpr(0xc00052cc40, 0x8b4860, 0xc0000def80)
        /home/neverik/go/src/github.com/go-critic/go-critic/checkers/flagName_checker.go:39 +0x10d
github.com/go-lintpack/lintpack/astwalk.(*exprWalker).WalkFile.func1(0x8b1ba0, 0xc0000def80, 0x1)
        /home/neverik/go/src/github.com/go-lintpack/lintpack/astwalk/expr_walker.go:23 +0x82
go/ast.inspector.Visit(0xc000343090, 0x8b1ba0, 0xc0000def80, 0x8b1160, 0xc000343090)
        /usr/local/go/src/go/ast/walk.go:373 +0x3a
go/ast.Walk(0x8b1160, 0xc000343090, 0x8b1ba0, 0xc0000def80)
        /usr/local/go/src/go/ast/walk.go:52 +0x66
go/ast.Walk(0x8b1160, 0xc000343090, 0x8b1ae0, 0xc000264360)
        /usr/local/go/src/go/ast/walk.go:147 +0x1897
go/ast.walkExprList(0x8b1160, 0xc000343090, 0xc000142d50, 0x1, 0x1)
        /usr/local/go/src/go/ast/walk.go:26 +0x9e
go/ast.Walk(0x8b1160, 0xc000343090, 0x8b1ba0, 0xc0000defc0)
        /usr/local/go/src/go/ast/walk.go:137 +0x1074
go/ast.Walk(0x8b1160, 0xc000343090, 0x8b1e60, 0xc000142d70)
        /usr/local/go/src/go/ast/walk.go:196 +0x1b12
go/ast.walkStmtList(0x8b1160, 0xc000343090, 0xc000142d80, 0x1, 0x1)
        /usr/local/go/src/go/ast/walk.go:32 +0x9e
go/ast.Walk(0x8b1160, 0xc000343090, 0x8b1b20, 0xc000264390)
        /usr/local/go/src/go/ast/walk.go:224 +0x1b67
go/ast.Walk(0x8b1160, 0xc000343090, 0x8b2120, 0xc0000df000)
        /usr/local/go/src/go/ast/walk.go:231 +0x1c73
go/ast.Walk(0x8b1160, 0xc000343090, 0x8b2120, 0xc0000df040)
        /usr/local/go/src/go/ast/walk.go:233 +0x1ce6
go/ast.walkStmtList(0x8b1160, 0xc000343090, 0xc0000df080, 0x3, 0x4)
        /usr/local/go/src/go/ast/walk.go:32 +0x9e
go/ast.Walk(0x8b1160, 0xc000343090, 0x8b1b20, 0xc0002643c0)
        /usr/local/go/src/go/ast/walk.go:224 +0x1b67
go/ast.Walk(0x8b1160, 0xc000343090, 0x8b1f60, 0xc0000df0c0)
        /usr/local/go/src/go/ast/walk.go:275 +0x6cb
go/ast.walkStmtList(0x8b1160, 0xc000343090, 0xc0000df100, 0x4, 0x4)
        /usr/local/go/src/go/ast/walk.go:32 +0x9e
go/ast.Walk(0x8b1160, 0xc000343090, 0x8b1b20, 0xc0002643f0)
        /usr/local/go/src/go/ast/walk.go:224 +0x1b67
go/ast.Walk(0x8b1160, 0xc000343090, 0x8b1fa0, 0xc000264420)
        /usr/local/go/src/go/ast/walk.go:344 +0xd6d
go/ast.Inspect(0x8b1fa0, 0xc000264420, 0xc000343090)
        /usr/local/go/src/go/ast/walk.go:385 +0x4b
github.com/go-lintpack/lintpack/astwalk.(*exprWalker).WalkFile(0xc00052cc60, 0xc0000f5580)
        /home/neverik/go/src/github.com/go-lintpack/lintpack/astwalk/expr_walker.go:21 +0x135
github.com/go-lintpack/lintpack.(*Checker).Check(0xc0001ad500, 0xc0000f5580, 0xc0000ebec0, 0xc0001ad500, 0xc00031cf60)
        /home/neverik/go/src/github.com/go-lintpack/lintpack/lintpack.go:126 +0x46
github.com/go-lintpack/lintpack/linter/lintmain/internal/check.(*linter).checkFile.func1(0xc0000ebec0, 0xc0000f5580, 0xc00031b980, 0xf, 0xf, 0xa, 0xc0001ad500)
        /home/neverik/go/src/github.com/go-lintpack/lintpack/linter/lintmain/internal/check/check.go:144 +0x87
created by github.com/go-lintpack/lintpack/linter/lintmain/internal/check.(*linter).checkFile
        /home/neverik/go/src/github.com/go-lintpack/lintpack/linter/lintmain/internal/check/check.go:126 +0x13a
```

--- a/checkers/utils.go
+++ b/checkers/utils.go
@@ -10,7 +10,7 @@ import (
 
 // isStdlibPkg reports whether pkg is a package from the Go standard library.
 func isStdlibPkg(pkg *types.Package) bool {
-	return pkg.Path() == pkg.Name()
+	return pkg != nil && pkg.Path() == pkg.Name()
 }
 
 // isUnitTestFunc reports whether FuncDecl declares testing function.

